how-to/customize-workspace/client/src/platform.ts

```diff
@@ -3,10 +3,22 @@ import { init as workspacePlatformInit, BrowserInitConfig } from '@openfin/works
 import { getSettings, validateThemes } from "./settings";
 import { getActions } from './actions';
 import { getDefaultWindowOptions, overrideCallback } from './browser';
+import { salesForceConnect, SalesforceSettings, providerId as salesforceProviderId } from './salesforce';
 
 export async function init() {
+    const settings = await getSettings();
+
+    const salesForceIntegration = settings?.integrationProvider?.integrations?.find(i => i.id === salesforceProviderId);
```

Comment:
could we move this to the bootstrapper.ts instead of here. This is where a platform in general is registered where as bootstrapper is checking to see if features are enabled and then registering them.

in the bootstrapper.ts we could have a map of key : integration and value object with a register and unregister function that would have a single entry at the moment (salesforce).

it then loops integration entries from manifest checks:

* Is enabled - yes
* Is mapped - yes
* call register and pass data

I don't know if the salesforce has any clean up that is needed but that could be in the unregister function that could be called alongside the home, notifications and store cleanup in the bootstrapper. The salesforce.ts could expose the register/unregister function that does the connect/cleanup.

In the future the settings could provide the path to the integration module that should be imported and the interface could have register/unregister alongside functions needed for home support etc.

Reply:
This was where it was in the original SF demo, but sure I can move it.